### PR TITLE
EDU-1215 Room improvements

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1215

**Example with all types of rooms on Dashboard**

Screen Shot 2023-03-08 at 20 15 03

### CODE REVIEW

File: src/components/dashboard/rooms-tab.yml [hunk 1]
```diff
@@ -1,28 +1,13 @@
...
-    en: Member
-    de: Mitglied
-  sortedByOwner:
-    en: Sorted by owner
-    de: Nach Eigentümer sortiert
-  sortedByRole:
-    en: Sorted by role
-    de: Nach Rolle sortiert
+    en: Here you find all the rooms you have created and the rooms that were invited to.
+    de: Hier findest du alle Räume, die du erstellt hast und in denen du eingeladen wurdest.
```
Comment (ahelmberger) [inline src/components/dashboard/rooms-tab.yml:4]: .. und in die du eingeladen wurdest.

File: src/components/dashboard/rooms-tab.yml [hunk 2]
```diff
@@ -1,28 +1,13 @@
...
   ownedRoomsHeadline:
     en: Rooms that you created
     de: Räume, die du erstellt hast
   memberRoomsHeadline:
-    en: Rooms that you are a member of
-    de: Räume, in denen du Mitglied bist
+    en: Rooms that you were invited to
+    de: Räume, in denen du eingeladen wurdest
```
Comment (ahelmberger) [inline src/components/dashboard/rooms-tab.yml:10]: .. in die du ..

File: src/components/dashboard/rooms-tab.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/dashboard/rooms-tab.yml]: Thanks for correcting (and explaining to me). Changed.

File: src/components/dashboard/rooms-tab.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/dashboard/rooms-tab.yml]: Changed as well.

**Comment by ahelmberger** [comment]
There is one wording that I find sounds not that nice: the label "is collaborative" next to the checkbox. I'd suggest either one of the two changes:

* remove the tooltip and put the longer explanation directly next to the checkbox as a label, like e.g.: "Allow invited users to create and modify documents in this room" (optionally with a "but not delete" thrown in)
* keep the tooltip explanation and rephrase the label to
    * "collaborative (work) mode" (I know, sounds a bit technical)
    * "work collaboratively"

Watch think?

**Comment by adrianaluchian** [comment]
> There is one wording that I find sounds not that nice: the label "is collaborative" next to the checkbox. I'd suggest either one of the two changes:
> 
> * remove the tooltip and put the longer explanation directly next to the checkbox as a label, like e.g.: "Allow invited users to create and modify documents in this room" (optionally with a "but not delete" thrown in)
> * keep the tooltip explanation and rephrase the label to one of ...
>   
>   * "collaborative (work) mode" (I know, sounds a bit technical)
>   * "work collaboratively"
> 
> Watch think?

Either option is better than "Is collaborative" / "Ist Kollaborativ". Changed to "Collaborative mode" / "Kollaborativer Modus", as discussed.

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**